doc/manual/src/installation/upgrading.md

```diff
@@ -1,25 +1,37 @@
 # Upgrading Nix
 
-First find the name of the current channel:
+> **Note**
+>
+> These upgrade instructions apply for regular Linux distributions where Nix was installed following the [installation instructions in this manual](./index.md).
+
+First, find the name of the current [channel](@docroot@/command-ref/nix-channel) through which Nix is distributed:
 
 ```console
 $ nix-channel --list
 ```
 
-Will return:
+By default this should return an entry for Nixpkgs:
 
 ```console
-$ nixpkgs https://nixos.org/channels/nixpkgs-23.05
+nixpkgs https://nixos.org/channels/nixpkgs-23.05
 ```
-Remove old channel:
+
+Check which Nix version will be installed:
```

Comment:
It seems unclear why the nix version needs to be checked, is it because a "upgrade" might somehow lower the nix version?

Reply:
Yes, you should be aware which source you're getting Nix from. Software versions are a notion that only exists in users' heads.